Tests/MapboxMapsTests/Foundation/Mocks/MockStyleManager.swift

```diff
@@ -0,0 +1,343 @@
+import Foundation
+@testable import MapboxMaps
+@_implementationOnly import MapboxCommon_Private
+
+final class MockStyleManager: StyleManagerType, Stubbable {
+
+    func asStyleManager() -> StyleManager {
+        fatalError()
+    }
+
+    @Stubbed var stubStyleURI: String!
+    func getStyleURI() -> String {
+        stubStyleURI ?? ""
+    }
+
+    func setStyleURIForUri(_ uri: String) {
+        stubStyleURI = uri
+    }
```

Comment:
Interesting approach to treat the getter and setter like a property. A couple thoughts…

* Rather than using nil-coalescing, I would just make `stubStyleURI` non-optional an initialize it to `""`
* This new approach has some merit, but it is also inconsistent with our other mocks. We should think carefully about what introducing this new technique would mean for the maintainability of the project overall. Would we want to update other mocks to use the same technique? If not, what would the impact be over time for other developers who come into the project?